app/src/main/java/org/liberty/android/fantastischmemo/common/AnyMemoDBOpenHelper.java

```diff
@@ -378,6 +386,36 @@ private void addToMultipleChoiceList(Cursor c, List<MultipleChoiceCard> list) {
     }
 
     public void updateMultipleChoiceId(String newId, String oldId) {
-        db.execSQL("update " + MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME +" set id = " + newId + " where id = " + oldId);
+        Cursor res = db.rawQuery("SELECT * FROM " + MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME,null);
+        int value = res.getColumnIndex("id");
+        if (value != -1) {
+            db.execSQL("update " + MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME +" set id = " + newId + " where id = " + oldId);
+        } else {
+            db.execSQL("update " + MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME + " set _id = " + newId + " where _id = " + oldId);
+        }
```

Comment:
What is the difference between _id and id?

Reply:
unit tests look for the column "id" while the app itself looks for "_id" so i need to check for both. I dont why it does that. I even tried dropping the tables and re adding them

Reply:
Ew thats ugly. So much extra code for nothing lol

Reply:
yeah true. If you have any ideas on how I can minimize it let me know lol

Reply:
actually i think i know how i can minimize it check it out after